### DISCUSSION TITLE
serverless func work in vercel dev but fail in vercel deploy

### DISCUSSION THREAD

**Discussion by jojorb** [vercel/vercel#4794] [Help]
Hi I made a simple node serverless function inside api folder like so: 
`api/siret/[siret].js`

It should query a third party api from url argument: 
my-vercel-serverless.app/api/siret/**1234567**

it work well with `vercel dev` but when i deployed it with no `vercel.json` it fail (i put env variable in all branch)
I also try many different kind of `vercel.json` routing but i didn't find any good one or find a good help in the documentation.

[full code here](https://gist.github.com/RobyRemzy/97581491e9febb892b11cdb0be851dd9)
Is there anything special to put on vercel.json to make it work?
is this happen because i've got a heder in the axios.get ?
```js
Authorization: `Bearer ${INSEE_TOKEN}`
```
 
thank you very much.

**Comment by paulogdm** [comment]
Please share a full reproduction example. Everything in the description and in the code looks good so far.

**Reply by jojorb** [reply]
thanks @paulogdm here my [repository](https://github.com/RobyRemzy/test-vercel-slinsee) and the vercel app is failing at this address:
here i want info about "30427545600016"
https://test-vercel-slinsee.vercel.app/api/siret/30427545600016

in vercel dev it return me a json

```json
{
  "header": {
    "statut": 200,
    "message": "ok"
  },
  "etablissement": {
    "siren": "304275456",
    "and....": "info...."
  }
}
```

**Reply by jojorb** [reply]
ok, i just update and do:
- add a `vercelignore`
- vercel
- check on https://test-vercel-slinsee.vercel.app/api/siret/30427545600016
(not working because i'm looking at the prod branch! it was push on dev)
- vercel --prod

it's working? wow! what just happen a simple vercelignore ? any idea?
